stuff/pipelines/database/contest.py

```diff
@@ -44,12 +53,21 @@ def get_contests(
               date_upper_limit.replace(hour=23, minute=59, second=59))
     )
     data: List[ContestCertificate] = []
+    ranking: List[Ranking] = []
     for row in cur:
+        scoreboard = client.contest.scoreboard(
+            contest_alias=row['alias'],
+            token=row['scoreboard_url'])
+        for position in scoreboard.ranking:
+            ranking.append(Ranking(
+                username=position.username,
+                place=position.place))
```

Comment:
Acá para que pueda ser leído desde el callback, hay que convertir la tupla `Ranking` a diccionario:
```suggestion
            ranking.append(Ranking(
                username=position.username,
                place=position.place)._asdict())
```